nixos/tests/soft-serve.nix

```diff
@@ -0,0 +1,103 @@
+{ system ? builtins.currentSystem
+, config ? { }
+, pkgs ? import ../.. { inherit system config; }
+, ...
+}:
```

Comment:
Please use the new syntax

Reply:
What is the new syntax, and where is it described? Maybe I missed that part in the documentation?

Reply:
I think I confused things here